libvast/src/system/archive.cpp

```diff
@@ -86,7 +86,9 @@ archive(archive_type::stateful_pointer<archive_state> self, path dir,
               VAST_DEBUG(self, "dismisses query for inactive sender");
               return make_error(ec::no_error);
             }
-            std::vector<event> result;
+            using receiver_type = caf::typed_actor<
+              caf::reacts_to<table_slice_ptr>>;
+            auto dst = caf::actor_cast<receiver_type>(self->current_sender());
```

Comment:
I find `dst` for "destination" a bit generic. How about calling this variable `client` or `requester` for added semantics?